topics/contributing/tutorials/critical_theory/tutorial.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: tutorial_hands_on
+
+title: "On Critical Learning Theory: the Hidden Curriculum"
+level: Advanced
+subtopic: pedagogy
+tags:
+    - critical learning theory
+    - pedagogy
+    - philosophy
+time_estimation: 2h
+questions:
+- What is Critical Learning Theory about?
+- What is its relation with the Critical Theory developed by the Frankfurt School?
+- How is possible to apply the principles of Critical Learning Theory?
+objectives:
+- To introduce the historical origins of Critical Learning Theory.
+- Systematically present the main methods associated with Critical Learning Theory.
+contributions:
+  authorship:
+    - gallardoalba
+---
+
+> <warning-title>Recommended starting point</warning-title>
+> This training can be considered as the continuation of [Principles of learning and how they apply to training and teaching ]({% link topics/contributing/tutorials/learning-principles/tutorial.md %}). If you wish to have a panoramic view of the theoretical corpus in which the **Critical Learning Theory** is embedded, we recommend to start with the above-mentioned training.
```

Comment:
please add a `follow_up_training` to that tutorial and a requirements to this one like

```
requirements:
  - type: "internal"
    topic_name: contributing
    tutorials:
      - learning-principles
```

though this (and the mentioned tutorial) both feel more like they belong in the `teaching` topic (@bebatut what do you think)